html5tagger/builder.py

```diff
@@ -115,12 +120,14 @@ def __call__(self, *_inner_content, **_attrs):
             self._endtag_close()
         return self
 
-    def _(self, *_content):
+    def _(self, *_content: Any):
         """Append new content without closing the current tag."""
         for c in _content:
             if c is None:
                 continue
-            assert c is not self, "Cannot add document to itself. Use E.elemname for sub snippets."
+            assert (
+                c is not self
+            ), "Cannot add document to itself. Use E.elemname for sub snippets."
```

Comment:
Looks like black formatter breaking the code. Please use the original format.

Reply:
There are a few other places where it similarly breaks lines it thinks are overlong. I would prefer keeping all the formatting changes out of this PR, but especially assert need to stay oneliners because often tracebacks only show one line of code.

I am currently looking for options regarding formatting. One called Ruff, combined with longer line length, seems usable. With black you cannot simply increase the line length because then it reformats shorter lines to maximum length, not respecting any coder decisions on what is clearest to read.

Reply:
I have very little experience with Ruff so far, but it combines isort, flake8 and black style formatting in one package, supposedly being mostly compatible with those, and is supposed to avoid the forced reformatting of line lengths that black (by its internal design) suffers of.